packages/forgejo/build.sh

```diff
@@ -21,6 +21,7 @@ termux_step_make() {
 	cp -a "$TERMUX_PKG_SRCDIR" "$GOPATH"/src/forgejo.org/forgejo
 	cd "$GOPATH"/src/forgejo.org/forgejo
 
+	unset CGO_LDFLAGS
```

Comment:
Removing LDFLAGS will remove the runpath from forgejo binary. The following output is present in current forgejo version 1.21.5-0 but it is absent after this change.

```
$ readelf -d $PREFIX/bin/forgejo | grep runpath
  0x000000000000001d (RUNPATH)         Library runpath: [/data/data/com.termux/files/usr/lib:/data/data/com.termux/files/usr/lib:/data/data/com.termux/files/usr/lib:/data/data/com.termux/files/usr/lib:/home/builder/.termux-build/_cache/android-r26b-api-24-v1/lib]
```